src/main/java/com/controller/ProjectController.java

```diff
@@ -0,0 +1,95 @@
+package com.controller;
+
+import com.dto.ProjectDto;
+import com.model.Project;
+import com.service.ProjectService;
+import lombok.extern.log4j.Log4j2;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Log4j2
+@Controller
+@RequestMapping(value = "/project")
+public class ProjectController {
+    private final ProjectService projectService;
+
+    private final ModelMapper modelMapper;
+
+    @Autowired
+    public ProjectController(ProjectService projectService, ModelMapper modelMapper) {
+        this.projectService = projectService;
+        this.modelMapper = modelMapper;
+    }
+
+
+    @RequestMapping
+    public ModelAndView listProjects(ModelAndView model) throws IOException {
+        List<ProjectDto> projectDtos = projectService.findAll()
+                .stream().map(this::convertToDto).collect(Collectors.toList());
+        model.addObject("projectList", projectDtos);
+        model.setViewName("project/project");
+        log.info("Project list showed");
+        return model;
+    }
+
+    @RequestMapping(value = "/newProject", method = RequestMethod.GET)
+    public ModelAndView newProject(ModelAndView model) {
```

Comment:
Краще додатково вказати що ти повертаєш сторінку створення проекту, а не створюєш проект
```suggestion
    public ModelAndView getCreateProjectView(ModelAndView model) {
```

Reply:
I see your point, thx